### DISCUSSION TITLE
Reduce space between code and output

### DISCUSSION THREAD

**Discussion by dvats** [quarto-dev/quarto-cli#1573] [Q&A]
How can I change the space between the command and the output? There is too much space in the default document that starts up!

<img width="656" alt="Screenshot 2022-07-29 at 8 34 14 AM" src="https://user-images.githubusercontent.com/6656840/181674317-5d6d0e72-89f5-4cfa-928f-095fd21b18f1.png">

**Comment by rgaiacs** [comment]
If you are exporting to HTML, you can use CSS.

`mwe.qmd`

``````
---
title: "MWE"
format:
  html:
    css: mwe.css
editor: visual
---

Some text.

```{r}
1 + 1
```

More text.
``````

`mwe.css`:

```
.cell-code {
    margin: 0 !important;
}
```

Be aware that the use of `!important;` in CSS should be avoid.

**Reply by dvats** [reply]
This is useful, however I am rending to pdf.

**Reply by rgaiacs** [reply]
PDF are built with LaTeX. Consider

``````
---
title: "MWE"
format:
  pdf:
    keep-tex: true
editor: visual
---

Some text.

```{r}
1 + 1
```

More text.
``````

Quarto will create an intermediate LaTeX file:

```
\begin{document}
\maketitle
\ifdefined\Shaded\renewenvironment{Shaded}{\begin{tcolorbox}[interior hidden, boxrule=0pt, borderline west={3pt}{0pt}{shadecolor}, sharp corners, enhanced, frame hidden, breakable]}{\end{tcolorbox}}\fi

Some text.

\begin{Shaded}
\begin{Highlighting}[]
\DecValTok{1} \SpecialCharTok{+} \DecValTok{1}
\end{Highlighting}
\end{Shaded}

\begin{verbatim}
[1] 2
\end{verbatim}

More text.



\end{document}
```

To change the space between the command and the output you will have to redefine the `Shaded` and `verbatim` LaTeX environment. If you are new to LaTeX, I would recommend that you make your peace with the default space because change things might have side effects. If you are a bit familiar with LaTeX, you can use https://tex.stackexchange.com/q/184019/22451 as a start point.

**Reply by dvats** [reply]
Thanks, I guess I'll make my peace with the spacing for now.
